mordecai3/geoparse.py

```diff
@@ -459,9 +459,10 @@ def geoparse_doc(self,
                 if len(scores) == 0:
                     logger.debug("No scores found.")
                     continue
-                if np.argmax(scores) == len(scores) - 1:
-                    logger.debug("Picking final ''null'' result.")
-                    continue
+                    # Commenting out the next 3 lines because of #https://github.com/semandex/mordecai3/issues/13
+                # if np.argmax(scores) == len(scores) - 1:
+                #     logger.debug("Picking final ''null'' result.")
+                #     continue
```

Comment:
My understanding is this issue was fixed [here](https://github.com/ahalterman/mordecai3/issues/6#issuecomment-1587703048). Was the error reintroduced?

Reply:
Yes, it got reintroduced later on with different intention